<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Паспорт скважины</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>500</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="name_wells">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Наименование скважины</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="wells_tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>410</width>
     <height>451</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_data">
    <attribute name="title">
     <string>Параметры</string>
    </attribute>
    <widget class="QLabel" name="alt_wells">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Абсолютная отметка
устья скважины</string>
     </property>
    </widget>
    <widget class="QLabel" name="depth_wells">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Глубина скважины</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="coord_wells_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>385</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Координаты скважины</string>
     </property>
     <widget class="QLineEdit" name="lat_lineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="y_wells">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Широта</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="long_lineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="x_wells">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Долгота</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="debit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Дебит</string>
     </property>
    </widget>
    <widget class="QLabel" name="decrease">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>245</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Понижение</string>
     </property>
    </widget>
    <widget class="QLabel" name="stat_lev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Статический уровень</string>
     </property>
    </widget>
    <widget class="QLabel" name="dinam_lev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Динамический уровень</string>
     </property>
    </widget>
    <widget class="QLabel" name="work_diam">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Рабочий диаметр 
колонны</string>
     </property>
    </widget>
    <widget class="QLabel" name="temp_c">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Температура воды</string>
     </property>
    </widget>
    <widget class="QLabel" name="data_beg">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата начала
бурения</string>
     </property>
    </widget>
    <widget class="QLabel" name="data_end">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>200</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата окончания
бурения</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="decrease_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="debit_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="stat_lev_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>270</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dinam_lev_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>300</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="temp_c_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>330</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="work_diam_spinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>180</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="begin_dateEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="end_dateEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>240</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="depth_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="abs_alt_doubleSpinBox">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_litol">
    <attribute name="title">
     <string>Литология</string>
    </attribute>
    <widget class="QComboBox" name="litol_comboBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>310</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="add_litol_pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
     <property name="icon">
      <iconset theme="add">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="del_litol_pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="litol_tableWidget">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>395</width>
       <height>291</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>125</number>
     </attribute>
     <column>
      <property name="text">
       <string>Глубина
залегания</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Описание</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>l_code</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_filters">
    <attribute name="title">
     <string>Фильтры</string>
    </attribute>
    <widget class="QTableWidget" name="filters_tableWidget">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>395</width>
       <height>371</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>90</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>25</number>
     </attribute>
     <row>
      <property name="text">
       <string>Начало</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Окончание</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>ph</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Fe</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Cl²</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>SO4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>NO3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>NO2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>CO3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>HCO3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>F</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>I</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Br</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>U</string>
      </property>
     </row>
    </widget>
    <widget class="QPushButton" name="add_filter_pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del_filter_pushButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>390</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_age">
    <attribute name="title">
     <string>Возраст</string>
    </attribute>
    <widget class="QTableWidget" name="age_tableWidget">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>395</width>
       <height>291</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>187</number>
     </attribute>
     <column>
      <property name="text">
       <string>Глубина</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Возраст</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="add_age_pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="del_age_pushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QComboBox" name="age_comboBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>310</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_adm">
    <attribute name="title">
     <string>Админка</string>
    </attribute>
    <widget class="QPushButton" name="clearTable_pushButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистка
таблицы</string>
     </property>
    </widget>
    <widget class="QPushButton" name="creat_db_pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Создать базу</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
